src/root.scss

```diff
@@ -0,0 +1,21 @@
+@import "carbon-components/scss/globals/scss/typography.scss";
+
+$ui-background: #ffffff;
```

Comment:
had to transfer some of the sidebar styles to this repo as well. a little redundancy...